src/core/colorspace.js

```diff
@@ -284,6 +284,17 @@ var ColorSpace = (function ColorSpaceClosure() {
           var stream = xref.fetchIfRef(cs[1]);
           var dict = stream.dict;
           numComps = dict.get('N');
+          alt = dict.get('Alternate');
+          if (alt) {
+            var altIR = ColorSpace.parseToIR(alt, xref, res);
+            // Parse the /Alternate CS to ensure that the number of components
+            // are correct, and that it is not a PatternCS.
```

Comment:
I'm going to push a new version of the patch, which slightly improves this comment. No functional code changes are made, though!